cmd/snap/cmd_snap_op.go

```diff
@@ -50,6 +50,26 @@ var (
 	pollTime    = 100 * time.Millisecond
 )
 
+type waitMixin struct {
+	NoWait bool `long:"no-wait" hidden:"true"`
+}
+
+// TODO: use waitMixin outside of cmd_snap_op.go
+
+var waitDescs = mixinDescs{
+	"no-wait": i18n.G("Do wait for the op to finish but just print the change id."),
```

Comment:
Shold this read `Do *not* wait for the to op to finish and just print the change id`?